src/pages/index.module.css

```diff
@@ -8,18 +8,76 @@
   text-align: center;
   position: relative;
   overflow: hidden;
+  background-color: #205493;
+  height: 65vh;
 }
 
-[data-theme="dark"] .heroTextColor {
-  color: white;
+.heroBanner::before {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-image: url("../../static/img/hero-banner.webp");
+  background-size: cover;
+  background-position: bottom;
+  opacity: 0.4;
+}
+
+.heroContainer {
+  position: relative;
+  z-index: 1;
+}
+
+.heroHeading {
+  font-size: 2.5rem;
+}
+
+.heroSubHeading {
+  font-size: 1.2rem;
+}
+
+.heroTextColor {
+  color: #fff;
```

Comment:
When color values map to specific color, the color name should be used.
`color: white;`

Reply:
Changes made here: https://github.com/NMFS-RADFish/radfish/pull/106/commits/dc9b9cc887c7dbb2c5be6fa2aa397ac8d20aa7a7